<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>291</width>
      <height>911</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Visualizacao</string>
     </attribute>
     <widget class="QRadioButton" name="isomerico">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>241</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Isomérico</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="ortogonal">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Navegação</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>30</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Modo de Visualização</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Câmera</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>241</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Posição Inicial :</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbAxis">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Exibir Eixos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxCameraX">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>69</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxCameraY">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>69</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxCameraZ">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>69</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QPushButton" name="posicionarCamera">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>261</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string extracomment="teste"/>
      </property>
      <property name="text">
       <string>Posicionar Camera</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="topo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Topo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cena</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <zorder>isomerico</zorder>
     <zorder>ortogonal</zorder>
     <zorder>label_11</zorder>
     <zorder>line_2</zorder>
     <zorder>label_16</zorder>
     <zorder>line_3</zorder>
     <zorder>label_17</zorder>
     <zorder>cbAxis</zorder>
     <zorder>label_18</zorder>
     <zorder>spinBoxCameraX</zorder>
     <zorder>label_19</zorder>
     <zorder>spinBoxCameraY</zorder>
     <zorder>label_20</zorder>
     <zorder>posicionarCamera</zorder>
     <zorder>spinBoxCameraZ</zorder>
     <zorder>topo</zorder>
     <zorder>label_22</zorder>
     <zorder>line_4</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Elementos</string>
     </attribute>
     <widget class="QPushButton" name="deletar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>730</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Deletar</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxH">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxW">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxL">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Altura</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Largura</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Comprimento</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>46</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>46</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>46</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxBlue">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>430</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxGreen">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxRed">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>370</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxX">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxY">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxZ">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <item>
       <property name="text">
        <string>Novo Elemento</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mesa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cama</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cadeira</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaveteiro</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>251</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Posição :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dimensão :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cor :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>67</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>10</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar Elemento</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="posicionar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>690</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Posicionar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adicionar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>650</y>
        <width>251</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotação :</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxRotation">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>480</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>90</number>
      </property>
     </widget>
     <zorder>deletar</zorder>
     <zorder>spinBoxH</zorder>
     <zorder>spinBoxL</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>spinBoxBlue</zorder>
     <zorder>spinBoxGreen</zorder>
     <zorder>spinBoxRed</zorder>
     <zorder>spinBoxX</zorder>
     <zorder>spinBoxY</zorder>
     <zorder>spinBoxZ</zorder>
     <zorder>label_13</zorder>
     <zorder>label_14</zorder>
     <zorder>label_15</zorder>
     <zorder>comboBox</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>label_12</zorder>
     <zorder>line</zorder>
     <zorder>posicionar</zorder>
     <zorder>spinBoxW</zorder>
     <zorder>adicionar</zorder>
     <zorder>label_21</zorder>
     <zorder>spinBoxRotation</zorder>
     <zorder>painelGL</zorder>
    </widget>
   </widget>
   <widget class="MeuPainelOpenGL" name="painelGL" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>881</width>
      <height>811</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>510</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>2000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOPen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuOPen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MeuPainelOpenGL</class>
   <extends>QWidget</extends>
   <header location="global">MeuPainelOpenGL.h</header>
   <container>1</container>
   <slots>
    <slot>alterarLados(int)</slot>
    <slot>alterarRaio(double)</slot>
    <slot>setRed(int)</slot>
    <slot>setGreen(int)</slot>
    <slot>setBlue(int)</slot>
    <slot>updateGL()</slot>
    <slot>updateOverlayGL()</slot>
    <slot>update()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
